src/overview.xml

```diff
@@ -106,7 +106,7 @@ Traffic parameters |            | Statistical criteria
       bottlenecks would then be permitted to contribute a larger
       fraction of the end-to-end
       packet loss budget. In compensation,
-      subpaths that to not exhibit bottlenecks have must be constrained to contribute less packet
+      subpaths that to not expected exhibit bottlenecks must be constrained to contribute less packet
```

Comment:
The comment crossed in the mail concurrent with the -10 draft posting and IETF LC.   I made the changes post -10, but don't know how they will go into the RFC editor.